DEPLOY.md

```diff
@@ -24,3 +30,6 @@ bundle exec pod trunk push --allow-warnings --verbose
 bundle exec fastlane deploy_ghpages ghpages_url:"https://github.com/rakutentech/ios-inappmessaging" github_token:<GitHub personal access token>
 ```
 
+#### Using tag
+
+By pushing a tag (example: 7.2.0), a Bitrise deploy workflow is triggered.
```

Comment:
The thing is that pushing a tag should be a required step in every method. The problem is that pushing a tag triggers Bitrise `deploy` workflow by default. Maybe we can add "Push a tag" point as a first point under "Deploy". Something like this:
```
## Deploy

Push a tag (example: 7.2.0)

### Using CI

Run `deploy` lane on Bitrise. (Should be automatically triggered after pushing a tag)

### Using local script
(in case Bitrise workflow failed or wasn't triggered)

Run following commands:
...
```

Reply:
@imaginaris Ok done.